tests/torch/runall.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Python virtual environment
+PYTHON_VENV="../../.venv-pytorch"
+
+# Clear SLURM logs (*.out and *.err files)
+rm -rf logs_slurm
+mkdir logs_slurm
```

Comment:
would `mkdir -p logs_slurm` not work here? If any of the files already exist then they will be overwritten anyway, and the rm will throw an error (though not terminate the program so that's okay I guess) if this folder doesn't already exist.

Reply:
this will not delete the directory. Anyways, this script is not so important